src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/updatecontactdetails/UpdateContactDetailsAboutToSubmitHandler.java

```diff
@@ -63,21 +65,12 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
             .map(wrapper -> wrapper.getUpdateIncludesRepresentativeChange() == YesOrNo.YES)
             .orElse(false);
 
-        YesOrNo isRespondentAddressHidden = contactDetailsWrapper
```

Comment:
update log message as you made it to support both case types.

Reply:
Thanks.  Great spot.